pkgs/applications/misc/gnss-sdr/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchFromGitHub
+, armadillo
+, blas
```

Comment:
Closer look also reveals that the same goes probably for `gfortran` and `-D GFORTRAN=YES`